### DISCUSSION TITLE
请问在测试时 打开首页 每次是默认显示文章目录  怎么才可以默认显示站点概览

### DISCUSSION THREAD

**Discussion by MuXiongGuo** [hugo-next/hugo-theme-next#64] [Q&A] [answered]
请问在测试时 打开首页 每次是默认显示文章目录  怎么才可以默认显示站点概览

**Comment by elkan1788** [comment, marked as answer]
把 `config.yaml` 配置文件中的 `toc--> enable` 设置为 `false` 就可以正常显示站点概览。

**Reply by MuXiongGuo** [reply]
谢谢!!!

**Reply by elkan1788** [reply]
客气啦，最新的开发版本 `4.4.0` 也针对此问题做了更多的优化，可以尝试升级到最新的开发版本。 😄
